content/en/docs/concepts/signals/logs.md

```diff
@@ -12,16 +12,17 @@ well-known, widely used logging libraries.
 
 ## OpenTelemetry logs
 
-OpenTelemetry does not define a bespoke API or SDK to create logs. Instead,
-OpenTelemetry logs are the existing logs you already have from a logging
-framework or infrastructure component. OpenTelemetry SDKs and
-autoinstrumentation utilize several components to automatically correlate logs
-with [traces](../traces).
-
-OpenTelemetry's support for logs is designed to be fully compatible with what
-you already have, providing capabilities to wrap those logs with additional
-context and a common toolkit to parse and manipulate logs into a common format
-across many different sources.
+OpenTelemetry provides a Logs API and SDK for producing log records, and
+language SDKs and autoinstrumentation integrate with existing logging
+frameworks. Log records in OpenTelemetry are modeled as events: a timestamped
+occurrence with a body and a set of attributes. The Logs API is public and can
+be used directly by application code or indirectly via existing logging
+libraries and bridges.
+
+OpenTelemetry is designed to work with the logs you already produce, offering
+tools to correlate logs with [traces](../traces), add contextual attributes, and
```

Comment:
Correlation works (or should work) with all other signals, not only traces, although it's the most prominent use case